packages/store/src/utils/app-studio.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @todo: This needs to be replaced by the functions from `@sap-ux/btp-utils`
+ */
+
+/**
+ * Enumeration of environment variables used in AppStudio
+ */
+ export enum ENV {
+    PROXY_URL = 'HTTP_PROXY',
+    H2O_URL = 'H2O_URL'
+}
+
+export function isAppStudio(): boolean {
+    return !!process.env[ENV.H2O_URL];
+}
+
+// export function getAppStudioBaseURL(): string | undefined {
+//     return process.env[ENV.H2O_URL];
+// }
+
+// export function getAppStudioProxyURL(): string | undefined {
+//     return process.env[ENV.PROXY_URL];
+// }
```

Comment:
Remove commented code?

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/237/commits/e37e1de9ea46e33d3a9e87c4febfd4187ff5186a